openl3/core.py

```diff
@@ -141,21 +209,26 @@ def get_audio_embedding(audio, sr, model=None, input_repr="mel256",
         Hop size in seconds.
     batch_size : int
         Batch size used for input to embedding model
+    frontend : str
+        The audio frontend to use. Can be 'kapre', 'librosa', 'auto', or None. 
+        If 'auto', it will determine the appropriate frontend based on the model input shape.
+        If None, you must provide an appropriate spectrogram of shape=(batch, time, freq, channel).
```

Comment:
I agree with this - it sounds like a better option

Reply:
I think this would be great and maybe we can add it in later. I'm always for having a fallback that gives you full flexibility (so you could use any frontend you want), but I don't think I really feel like implementing this now lolol.